src/utilities/use-deep-callback.tsx

```diff
@@ -0,0 +1,11 @@
+import {useCallback} from 'react';
```

Comment:
I know these are pretty simple but I wonder if adding tests would be worth it. If anything for documentation purposes. What do you think?

Reply:
In general, the approach to testing hooks is by the implementation, rather than internals. So using them in a component, and testing the component. That's not to say we shouldn't test hooks, at least in this situation the hook in so minimal we would almost be testing react internals. If you have a suggestion my ears are open 😄  Right now web is using the test by implementation approach, and not specifically testing hooks. We could always spend some time looking at testing approaches.

Reply:
I find the tests they have in quilt useful because they clearly demonstrate what is expected of the hook.